db/migrate/20180409120422_create_physical_network_ports.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePhysicalNetworkPorts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :physical_network_ports do |t|
+      t.bigint  :ems_ref
+      t.string  :uid_ems
+      t.string  :port_name
+      t.string  :port_type
+      t.string  :peer_mac_address
+      t.string  :vlan_key
+      t.string  :address
```

Comment:
Does it make sense that a port would have an address?  This is already on the GuestDevice model and I think of a port as a layer 1 concept and mixing address pulls in layer 3

Reply:
Actually this field is about the port mac address, so I think that is better to change the field name to `mac_address`